config/vendor-api-v1.yml

```diff
@@ -238,6 +239,45 @@ paths:
           $ref: "#/components/responses/OK"
         "401":
           $ref: "#/components/responses/Unauthorized"
+  "/test-data/generate":
```

Comment:
Do we add this now, since it's not available without the feature flag? (If we do, should we add the feature flag?

Reply:
Right, this shouldn't be in here until it's ready to release. Will remove for now